libxkb will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libxkb it will fail in the following way:

```
2024/11/12 23:12:50 INFO running step "test/pkgconf"
2024/11/12 23:12:50 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:12:50 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:12:50 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:12:50 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:12:50 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + basename /home/build/melange-out/libxkb-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + dev_pkg=libxkb-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + apk info -L libxkb-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + grep -q ^Name: usr/lib/pkgconfig/xkbfile.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + basename usr/lib/pkgconfig/xkbfile.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + lib_name=xkbfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + echo usr/lib/pkgconfig/xkbfile.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + grep -q '^usr/lib/pkgconfig/xkbfile.pc$\|^usr/share/pkgconfig/xkbfile.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + pkgconf --exists xkbfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN + pkgconf --print-errors --exists xkbfile uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN Package kbproto was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN Perhaps you should add the directory containing `kbproto.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 WARN Package 'kbproto', required by 'xkbfile', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:12:50 INFO pod 5dbecef9222dc68aeadf84495480a21d9436144ede82219c6d9176b8bbe82fe7 terminated
2024/11/12 23:12:50 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:12:50 INFO   workspace dir: /home/user/tmp/melange-workspace-3062373486
2024/11/12 23:12:50 INFO   guest dir: /home/user/tmp/melange-guest-20912166
2024/11/12 23:12:50 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libxkb] Error 1
```

--- a/libxkb.yaml
+++ b/libxkb.yaml
@@ -1,7 +1,7 @@
 package:
   name: libxkb
   version: 1.1.3
-  epoch: 0
+  epoch: 1
   description: "X11 keyboard file manipulation library"
   copyright:
     - license: MIT
@@ -35,8 +35,14 @@ pipeline:
 subpackages:
   - name: "libxkb-dev"
     description: "headers for libxkb"
+    dependencies:
+      runtime:
+        - libx11-dev
     pipeline:
       - uses: split/dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

